cpp/include/cuml/fil/multi_reduction.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
```

Comment:
nitpick: since this is a header file with device-side methods, rename this to `.cuh` to follow similar convention done at other places in cuML. (ref: #1675)